OpenRA.Game/Game.cs

```diff
@@ -352,7 +317,54 @@ public static void InitializeMod(string mod, Arguments args)
 				Environment.Exit(0);
 			}
 			else
+			{
```

Comment:
If Environment.Exit works the way I assume it does (terminating the application) then you could keep the following in the main flow instead of inside the else.  That would minimise the future change when the dedicated server code is pulled into its own executable.